Snowflake.Data/Core/FileTransfer/StorageClient/SFS3Client.cs

```diff
@@ -37,16 +37,16 @@ internal class S3Metadata
         /// The metadata header keys.
         /// </summary>
         private const string AMZ_META_PREFIX = "x-amz-meta-";
-        private const string AMZ_IV = "x-amz-iv";
-        private const string AMZ_KEY = "x-amz-key";
-        private const string AMZ_MATDESC = "x-amz-matdesc";
-        private const string SFC_DIGEST = "sfc-digest";
+        internal const string AMZ_IV = "x-amz-iv";
+        internal const string AMZ_KEY = "x-amz-key";
+        internal const string AMZ_MATDESC = "x-amz-matdesc";
+        internal const string SFC_DIGEST = "sfc-digest";
 
         /// <summary>
         /// The status of the request.
         /// </summary>
-        private const string EXPIRED_TOKEN = "ExpiredToken";
-        private const string NO_SUCH_KEY = "NoSuchKey";
+        internal const string EXPIRED_TOKEN = "ExpiredToken";
```

Comment:
I will create a separate ticket for non-dotnet notation for entire project

Reply:
Ok